File: tech-studio-projects/other-projects/document-management-system/java/src/main/java/com/dms/UserManager.java
package com.dms;

import java.util.ArrayList;
import java.util.List;

public class UserManager {
    private List<User> users;

    public UserManager() {
        users = new ArrayList<>();
        users.add(new User("admin", "1234")); // Default user
    }

    public User getUserByUsername(String username) {
        for (User user : users) {
            if (user.getUsername().equals(username)) {
                return user;
            }
        }
        return null;
    }
}

File: tech-studio-projects/other-projects/document-management-system/csharp/csharp/UserManager.cs
using System.Collections.Generic;
using System.Linq;

namespace DocumentManagementSystem
{
    public class UserManager
    {
        private readonly List<User> users;

        public UserManager()
        {
            users = new List<User>
            {
                new User("admin", "1234") // Default user
            };
        }

        public User GetUserByUsername(string username)
        {
            return users.FirstOrDefault(user => user.Username == username);
        }
    }
}
